IRC.md

```diff
@@ -1,111 +1,238 @@
 # Connecting to IRC
 
-Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are some cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
+Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are **several** cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
 
 Lines prefixed with < are sent from client to server, and lines prefixed with > are sent from the server to the connecting client.
 
 ## Prerequisites
+
 In order to connect to Twitch IRC, you must have three pieces of information:
 
 1. The name of channel that you want to join.
 2. A Twitch account.
-3. Oauth token from API or from a site using it: www.twitchapps.com/tmi 
+3. An OAuth token [authorized through our API](/authentication.md) either directly or via an application which provides it
```

Comment:
Is there a reason to no longer link to twitchapps.com/tmi? It seems very useful for newcomers.